docs/user/convert/function_usage.rst

```diff
@@ -679,6 +679,18 @@ Example: ``stringToBoolean('true', false) = true``
 
 Example: ``stringToBoolean('55', false) = false``
 
+intToBoolean
+^^^^^^^^^^^^^^^
```

Comment:
the `^`s here need to match the length of the title, otherwise the sphinx build will complain.